lib/index.ts

```diff
@@ -68,7 +68,12 @@ export class Compiler {
       case ts.SyntaxKind.RestType: return this._compileRestTypeNode(node as ts.RestTypeNode);
       case ts.SyntaxKind.UnionType: return this._compileUnionTypeNode(node as ts.UnionTypeNode);
       case ts.SyntaxKind.IntersectionType: return this._compileIntersectionTypeNode(node as ts.IntersectionTypeNode);
-      case ts.SyntaxKind.LiteralType: return this._compileLiteralTypeNode(node as ts.LiteralTypeNode);
+      case ts.SyntaxKind.LiteralType:
+        let tnode = node as ts.LiteralTypeNode
+        if (tnode.literal.kind === ts.SyntaxKind.NullKeyword) {
```

Comment:
This would be a bit cleaner if moved into `_compileLiteralTypeNode` helper (where `node` is already known to have the right type)